CHANGELOG

```diff
@@ -99,7 +99,7 @@ qterminal-0.9.0 / 2018-05-21
   * Fix tabstop order in properties dialog & add buddy relations for labels. (#290)
   * Add preferences for background images (#273)
   * Disable menu actions that are not applicable currently (#244)
-  * DEFINES += STR_VERSION=\\\"0.7.0\\\"
+  * DEFINES += QTERMINAL_VERSION=\\\"0.7.0\\\"
```

Comment:
Probably you automatically changed all strings. In the ChangeLog it shouldn't be changed however.